lib/chef/resource/aws_cache_subnet_group.rb

```diff
@@ -0,0 +1,22 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+class Chef::Resource::AwsCacheSubnetGroup < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::ElastiCache, id: :group_name
```

Comment:
Yes, it didn't work without this. It gave me an error about the 'name' attribute.